fvtest/compilertriltest/CMakeLists.txt

```diff
@@ -39,6 +39,7 @@ add_executable(comptest
 	ShiftAndRotateTest.cpp
 	SimplifierFoldAndTest.cpp
 	IfxcmpgeReductionTest.cpp
+        LongAndAsRotateTest.cpp
```

Comment:
I think the CMakeLists.txt files use tabs, hence the spacing inconsistency here. Can we modify the spaces to tabs?